calico/felix/fetcd.py

```diff
@@ -731,10 +771,6 @@ def on_tiered_policy_delete(self, response, tier, policy_id):
         self.splitter.on_policy_selector_update(policy_id, None, None)
 
     def on_host_ip_set(self, response, hostname):
-        if not self._config.IP_IN_IP_ENABLED:
```

Comment:
I think youy should move this if test into `_update_hosts_ipset()` so that we update `self.ipv4_by_hostname` but don't actually program the dataplane.